resources/lib/omdb/mapping.py

```diff
@@ -30,7 +30,7 @@ def __init__(self):
                 'func': lambda v: get_between_strings(v or '', 'Won ', ' Golden Globe')}, {
                 # ---
                 'keys': [('infoproperties', 'award_wins')],
-                'func': lambda v: get_between_strings(v or '', '. Another ', ' wins') or get_between_strings(v or '', '', ' wins')}, {
+                'func': lambda v: get_between_strings(v or '', '. Another ', ' win') or get_between_strings(v or '', 'Oscars. ', ' win') or get_between_strings(v or '', 'Oscar. ', ' win') or get_between_strings(v or '', '', ' win')}, {
```

Comment:
`.` period is a special regex character (single any character wildcard) so you need to escape it here `'Oscar\. '` otherwise it will also match `Oscars` without the period.

In fact, can skip that altogether and simplify the whole line using `.* ` (i.e. any number of wildcard characters followed by a space).
`get_between_strings(v or '', '.* ', ' win') or get_between_strings(v or '', '', ' win')`